includes/Stories_Renderer/Renderer.php

```diff
@@ -512,4 +541,75 @@ protected function get_content_overlay() {
 
 	}
 
+	/**
+	 * Renders the lightbox markup for non-amp pages.
+	 *
+	 * @return void
+	 */
+	protected function render_stories_with_lightbox_noamp() {
+		$stories = $this->story_posts;
+		?>
+		<div class="web-stories-list__lightbox">
+			<amp-story-player width="0" height="0" layout="responsive">
+				<script type="application/json">
+					{
+						"controls": [
+							{
+								"name": "close",
+								"position": "start"
+							},
+							{
+								"name": "skip-next"
+							}
+						]
+					}
```

Comment:
```suggestion
		<?php
		         $data = [
		              'controls' => [
		                    [
		                       'name' =>  'close'
		                       'position' =>  'start'
		                    ],
		                    [
		                        'name' => 'skip-next'
		                    ],
		              ]
		         ];
		         
		         echo wp_json_encode( $data );
```

Would it be cleaner do this?